pkg/controller/drain/drain_controller.go

```diff
@@ -327,7 +327,7 @@ func (ctrl *Controller) syncNode(key string) error {
 	if err := ctrl.setNodeAnnotations(node.Name, annotations); err != nil {
 		return fmt.Errorf("node %s: failed to set node uncordoned annotation: %w", node.Name, err)
 	}
-
+	ctrlcommon.UpdateStateMetric(ctrlcommon.MCCSubControllerState, "machine-config-controller-drain", "Sync Node", node.Name)
```

Comment:
Since this only applies to when a successful cordon+drain or a successful uncordon happens, should we somehow indicate that? (instead of just Sync Node, unless that's a specific value you want?)